assets/js/player.js

```diff
@@ -59,6 +49,38 @@ videojs.Vhs.xhr.beforeRequest = function(options) {
 
 var player = videojs('player', options);
 
+/**
+ * Function for add time argument to url
+ * @param {String} url 
+ * @returns urlWithTimeArg
+ */
+function addCurrentTimeToURL(url) {
+    var urlUsed = new URL(url);
+    urlUsed.searchParams.delete('start');
+    var currentTime = Math.ceil(player.currentTime());
+    if (currentTime > 0)
+        urlUsed.searchParams.set('t', currentTime);
+    else
+        if (urlUsed.searchParams.has('t'))
+            urlUsed.searchParams.delete('t');
+    return urlUsed;
+}
+
+var shareOptions = {
+    socials: ['fbFeed', 'tw', 'reddit', 'email'],
+
+    get url() {
+        return addCurrentTimeToURL(short_url);
+    },
+    title: player_data.title,
+    description: player_data.description,
+    image: player_data.thumbnail,
+    get embedCode() {
+        return "<iframe id='ivplayer' width='640' height='360' src='" +
+            addCurrentTimeToURL(embed_url) + "' style='border:none;'></iframe>";    
```

Comment:
There are extra spaces at the end of the line, btw!

Reply:
I'm cleaning this up!